presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -734,6 +736,7 @@ public void close()
         Footer footer = new Footer(
                 numberOfRows,
                 rowGroupMaxRowCount,
+                OptionalLong.of(rawSize),
```

Comment:
Please add a test for the file footer raw size.

Reply:
Done. (Also added a test for Stripe raw size as well)